lib/kinetic.jl

```diff
@@ -100,7 +113,8 @@ end
 
 ## Parses reactions of the form
 ##
-## :-> a b rate 
+## :-> a b rate  or :→ a b rate 
+## :⇄ a b rate1 rate2
 ##
 function parseReactionSystem (V)
```

Comment:
How about `parsereactions` for a name? I'm not fond of the capitalization here. 

Also, because it looks like this function is meant to be used by modelers, it should have a documentation string.